INIT.md

```diff
@@ -1,6 +1,6 @@
 # webpack-cli init
 
-`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration, generating some boilerplate and installing modules required for the project as per user preferences.
```

Comment:
B specific, better to tell them that we create a config and make a runnable config, not boilerplate